CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ cmake_dependent_option(USE_SSE "Build with x86 SSE instruction support" ON
   "CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL amd64" OFF)
 option(USE_F16C "Build with x86 F16C instruction support" ON) # autodetects support if ON
 option(USE_LAPACK "Build with lapack support" ON)
+option(USE_ILP64_LAPACKE "Use ILP64 LAPACKE interface" OFF)
+option(USE_LAPACKE_INTERFACE "Use LAPACKE interface for lapack support" OFF)
+option(LINK_GFORTRAN "Link to gfortran" OFF)
+option(LINK_GOMP "Link to gomp" OFF)
```

Comment:
Please be aware that gomp is just one omp implementation. For example, if user compiles with clang, llvm omp will be used. Same goes for intel compiler and iomp

Reply:
Ohh.. Right. I might build openblas without omp then. The apt get version of openblas is built without any omp

Reply:
Ubuntu offers multiple OpenBlas packages. Serial, pthread and openmp based. You can refer to https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=openblas&searchon=names

I'd guess that adopting serial OpenBlas is not an option due to potential slowdown. If you think it might be ok, it would be great to run a benchmark to check before making a decision.